manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -39,6 +39,14 @@ spec:
           for: 15m
           labels:
             severity: critical
+        - alert: PodsUnableToDelete
+          annotations:
+            summary: Some pods are unable to be deleted.
+            description: Some pods have been unable to be deleted for last 24h (since their grace period for termination has been reached).
```

Comment:
I think it would be best to introduce a runbook_url for this and keep it general here. For finding the pods, maybe we can just point them to the query without the `namespace` aggregation?